src/ModelContextProtocol/Utils/Json/CustomizableJsonStringEnumConverter.cs

```diff
@@ -1,91 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System.ComponentModel;
-using System.Diagnostics.CodeAnalysis;
-using System.Reflection;
-using System.Text.Json;
-using System.Text.Json.Serialization;
-
-// NOTE:
-// This is a workaround for lack of System.Text.Json's JsonStringEnumConverter<T>
-// 9.x support for JsonStringEnumMemberNameAttribute. Once all builds use the System.Text.Json 9.x
-// version, this whole file can be removed. Note that the type is public so that external source
-// generators can use it, so removing it is a potential breaking change.
-
-namespace ModelContextProtocol.Utils.Json;
-
-/// <summary>
-/// A JSON converter for enums that allows customizing the serialized string value of enum members
-/// using the <see cref="JsonStringEnumMemberNameAttribute"/>.
-/// </summary>
-/// <typeparam name="TEnum">The enum type to convert.</typeparam>
-/// <remarks>
-/// This is a temporary workaround for lack of System.Text.Json's JsonStringEnumConverter&lt;T&gt;
-/// 9.x support for custom enum member naming. It will be replaced by the built-in functionality
-/// once .NET 9 is fully adopted.
-/// </remarks>
-[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Why are the converters being deleted? It seems like an unrelated change.

Reply:
Dead code presumably?

Reply:
It's not deleted, the file just moved folders and GH isn't displaying it as a rename for some reason